src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $product-card-width;
+  background-color: #fff;
+  border: 1px solid $main-border-color;
```

Comment:
The variable `$main-border-color` is used here. Ensure this variable is defined in your SCSS files.